experiments/snippets/debugging/actionFramework/actionFrameworkDebugging.js

```diff
@@ -0,0 +1,34 @@
+/*
+Action logging in Skuid! This adds console output for all actions called during page runtime.
+Paste this into an 'Inline' or include it as a page resource.
```

Comment:
```suggestion
Paste this into an **Inline** resource or include it as a page resource.
```
I would also add a line stating that the **In-Line Resource** option _will not_ work. It took me a second to realize why the snippet wasn't working on my page.